### PR TITLE
add jquery-weui@0.8.0 w/ npm autoupdate

### LINKED ISSUE #8547: [Request] Add jquery-weui
**Library name: jquery-weui** 
**Git repository url: https://github.com/lihongxun945/jquery-weui**
**npm package url(optional):** 
**License(s): MIT License**
**Official homepage: http://lihongxun945.github.io/jquery-weui/**
**Wanna say something? Leave message here:**
jquery-weui is an UI for build hybrid web app in wechat - which is the most popular IM in china.
=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #8547 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/lihongxun945/jquery-weui
 * Official website (optional, not the repository): https://lihongxun945.github.io/jquery-weui/
 * NPM package url (optional): https://www.npmjs.com/package/jquery-weui
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1339
   - Count of watchers: 162
   - Count of forks: 401
 * NPM download stats (optional):
   - Downloads in the last day: 127
   - Downloads in the last week: 231
   - Downloads in the last month: 617

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8547, cc @lihongxun945

### CODE REVIEW

File: ajax/libs/jquery-weui/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "url": "https://github.com/lihongxun945/jquery-weui"
+  },
+  "homepage": "https://lihongxun945.github.io/jquery-weui/",
+  "license": "MIT",
+  "npmName": "jquery-weui",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "+(js|css)/*"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-weui/package.json]: `"+(js|css)/**/*"`

**Comment by pvnr0082t** [comment]
@x09326 I have modified auto-update config. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t have you tried to exclude the other folders?

**Comment by pvnr0082t** [comment]
@x09326 I have modified auto-update config using exclude. Please help me review again, thank you.

**Comment by lihongxun945** [comment]
Great work! Thank you :smile:
CDNJS is fast in China too

**Comment by PeterDaveHello** [comment]
@lihongxun945 Oh really?  Good news!

**Comment by lihongxun945** [comment]
Some user say it's blocked by GFW :sweat:
I think cndjs is only available in some area of China. I have find a available mirror for Chinese users http://www.bootcdn.cn/#about

**Comment by lihongxun945** [comment]
How can I donate cndjs? I could not find donate link in your website

**Comment by lihongxun945** [comment]
Done! $50 donated sweat: for this project @PeterDaveHello

**Comment by PeterDaveHello** [comment]
@lihongxun945 There is a "Support us" menu on homepage, would you please take a look to see if the methods can be good for you? Or maybe you will like to use paypal?

**Comment by PeterDaveHello** [comment]
@lihongxun945 Oh great, thanks for your donation.

**Pull request merged**